sound/hybrid_font.h

```diff
@@ -162,7 +167,7 @@ class HybridFont : public SaberBase {
     if(swing_player_) {
       if (swing_player_->isPlaying()) {
         float accent_volume = powf(swing_strength, config_.ProffieOSSwingVolumeSharpness) * config_.ProffieOSMaxSwingVolume;
-        swing_player_->set_fade_time(0.04);
+        swing_player_->set_fade_time(0.1);
```

Comment:
this is unrelated to the slash implementation

Reply:
yes, I'll revert.